code/modules/client/preference/loadout/loadout_uniform.dm

```diff
@@ -386,3 +386,24 @@
 	display_name = "Orange Hawaiian T-Shirt"
 	description = "A nice t-shirt to remind about warm beaches. This one is orange."
 	path = /obj/item/clothing/under/misc/orangehawaiianshirt
+
+/datum/gear/uniform/waiter
+	display_name = "Waiter's outfit"
+	path = /obj/item/clothing/under/misc/waiter
+	allowed_roles = list("Chef")
+
+/datum/gear/uniform/vice
+	display_name = "Vice officer's jumpsuit"
+	path = /obj/item/clothing/under/misc/vice
+
+/datum/gear/uniform/overalls
+	display_name = "Laborer's overalls"
+	path = /obj/item/clothing/under/misc/overalls
+
+/datum/gear/uniform/wizard
+	display_name = "Wizard uniform"
+	path = /obj/item/clothing/under/color/lightpurple
```

Comment:
We don't give out wizard stuff to crew, and we already have this path as loadout gear
```
/datum/gear/uniform/suit/jumpsuitlightpurple
	display_name = "Jumpsuit, lightpurple"
	path = /obj/item/clothing/under/color/lightpurple
```